docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -7,39 +7,27 @@ The main features of the Image Util API include:
 
 - Conversion
 
-  You can [change the color space format](#colorspace) among the [supported formats](#color_format).
+  You can change the [image](#image_colorspace) and [media packet](#mediapacket_colorspace) color space among the [supported formats](#color_format).
 
 - Resizing
 
-  You can [change the image resolution](#resize).
+  You can change the [image](#resize_image) and [media packet](#resize_mediapacket) resolution.
 
 - Rotation
 
-  You can [change the image angle](#rotate) around the X or Y axis.
+  You can change the [image](#rotate_image) and [media packet](#rotate_mediapacket) angle around the X or Y axis.
 
 - Crop
 
-  You can [remove the outer parts of an image](#crop) or change the aspect ratio.
+  You can crop the outer parts of the [image](#crop_image) and [media packet](#crop_mediapacket).
 
 - Decoding from a file or memory and encoding to a file or memory
 
   You can [decode images](#decode) and [encode them](#encode) with the following formats:
 
-  - Bitmap formats:
-
-    - YUV420, YUV422, RGB888, RGBA8888, BGRA8888, ARGB8888
-
-  - Input image formats for decoding:
-
-    - JPEG, PNG, GIF, BMP
-
-    > **Note**
-    >
-    > Animated GIF format is not supported for decoding.
-
-  - Output image formats for encoding:
-
-    - JPEG, PNG, GIF, BMP, [animated GIF](#animation)
+  - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888.
+  - Input image formats for decoding: JPEG, PNG, GIF, and BMP.
+  - Output image formats for encoding: JPEG, PNG, GIF, BMP, and [animated GIF](#animation).
 
     Pay attention to how the [image quality depends on the size](#quality) and compression ratio.
```

Comment:
This seems to be a note. Also, this sentenceis applicable for all the above 3 sub bulleted list. Request you to please left align it.

> **Note**
>
> Pay attention to how the [image quality depends on the size](#quality) and compression ratio.

Reply:
Hello, the sentence applicable for "decoding from a file *** memory".
I will left align it.

Reply:
@jiyong2, please remove the periods at the end of the sentence as it is not a complete sentence.

 - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
  - Input image formats for decoding: JPEG, PNG, GIF, and BMP
  - Output image formats for encoding: JPEG, PNG, GIF, BMP, and [animated GIF](#animation)